system/lib/wasmfs/js_api.cpp

```diff
@@ -290,6 +291,15 @@ int _wasmfs_mount(char* path, wasmfs::backend_t created_backend) {
 // directory existed before.
 int _wasmfs_unmount(char* path) { return wasmfs_unmount((intptr_t)path); }
 
+int _wasmfs_ioctl(int fd, int request, ...) {
```

Comment:
The caller of this method is in JS IIANM? If so it always passes in the same number of arguments, so `...` is not needed here (and adds significant complexity). Am I missing something?

Reply:
Ah, I completely overlooked this. I agree that `...` isn't needed.